tests/wpt/meta/css/css-grid/__dir__.ini

```diff
@@ -0,0 +1 @@
+prefs: ["layout.columns.enabled:true", "layout.flexbox.enabled:true", "layout.grid.enabled:true"]
```

Comment:
You have added the preference, but are you using it somewhere as well? Maybe I just missed it above.

Reply:
It sets a corresponding preference in Stylo. `display: grid` fails to parse if the Stylo pref isn't set, so none of the other code gets hit.